docs/examples/track_fusion_example.py

```diff
@@ -0,0 +1,538 @@
+from stonesoup.metricgenerator.manager import MultiManager
+
+# Lists to hold the detections from each sensor
+s1_detections = []
+s2_detections = []
+
+groundtruth_generator = groundtruth_simulation.groundtruth_paths_gen()
+
+truths = set()
+# Iterate over the time steps, extracting the detections and truths
+for (time, sd1), (_, sd2) in zip(radar1plot, radar2plot):
+    s1_detections.append(sd1)
+    s2_detections.append(sd2)
+    truths.update(next(groundtruth_generator)[1])  # consider only the path
+
+# Plot the detections from the two radars
+plotter = Plotterly()
+plotter.plot_measurements(s1_detections, [0, 2], marker=dict(color='orange', symbol='305'),
+                          measurements_label='Sensor 1 measurements')
+plotter.plot_measurements(s2_detections, [0, 2], marker=dict(color='blue', symbol='0'),
+                          measurements_label='Sensor 2 measurements')
+plotter.plot_sensors({sensor1_platform, sensor2_platform}, [0, 1],
+                     marker=dict(color='black', symbol='1', size=10))
+plotter.plot_ground_truths(truths, [0, 2])
+plotter.fig
+
+# %%
+# 2. Initialise the trackers components
+# -------------------------------------
+# We have initialised the sensors and the target trajectory, we can see that the
+# detections from the two sensors differ from one another, that will lead to two separate tracks.
+#
+# Now, we initialise the components of the two trackers, one using an Extended Kalman filter
+# and a particle filter.
+#
+
+# %%
+# Stone Soup tracker components
+# ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+#
+
+# Let's consider a distance based hypothesiser
+from stonesoup.hypothesiser.distance import DistanceHypothesiser
+from stonesoup.measures import Mahalanobis
+from stonesoup.dataassociator.neighbour import GNNWith2DAssignment
+from stonesoup.deleter.time import UpdateTimeStepsDeleter
+
+# Load the kalman filter components
+from stonesoup.updater.kalman import UnscentedKalmanUpdater
+from stonesoup.predictor.kalman import UnscentedKalmanPredictor
+from stonesoup.initiator.simple import SimpleMeasurementInitiator, GaussianParticleInitiator
+
+# Load a single target tracker
+from stonesoup.tracker.simple import SingleTargetTracker
+
+# prepare the particle components
+from stonesoup.predictor.particle import ParticlePredictor
+from stonesoup.updater.particle import ParticleUpdater
+from stonesoup.resampler.particle import ESSResampler
+
+
+# Let's define a helper function to minimise the number of times
+# we have to initialise the same tracker object
+def general_tracker(tracker_class, detector,
+                    filter_updater, initiator, deleter,
+                    data_associator):
+    """
+    Helper function to initialise the trackers
+    """
+
+    tracker = tracker_class(
+        initiator=initiator,
+        detector=detector,
+        updater=filter_updater,
+        data_associator=data_associator,
+        deleter=deleter)
+    return tracker
```

Comment:
This doesn't seem to have a purpose other than just passing arguments through? Assume this was for something that was later removed?